### PR TITLE
Adding and removing LDM+HVM instances with basic auth

### PR DESCRIPTION
**Pull request by benradstone**

### CODE REVIEW

File: docs/get-started.md [hunk 1]
```diff
@@ -4,27 +4,61 @@ title: Get started
 sidebar_label: Get started
 ---
 
-You can use either LiveData Migrator's UI or CLI to manage resources and migrate data. If you're using the [UI](#get-started-with-the-livedata-migrator-ui) to operate LiveData Migrator, you'll need to open ports and register your LiveData Migrator account after installation. If you're using the [CLI](#get-started-with-the-livedata-migrator-cli) to operate LiveData Migrator, you're ready to log in.
+You can use the LiveData Migrator UI or CLI to manage resources and migrate data.
 
-## Get started with the LiveData Migrator UI
+If you want to use the [UI](#get-started-with-the-livedata-migrator-ui) to operate LiveData Migrator, register your LiveData Migrator account after installation. If you want to use the [CLI](#get-started-with-the-livedata-migrator-cli) to operate LiveData Migrator, you're ready to log in.
```
Comment (stepgerner) [inline docs/get-started.md:9]: Minor nitpick, suggest making the URLs encapsulate the entire "use the UI to operate LiveData Migrator" sentence and "use the CLI to operate LiveData Migrator" respectively, so they flow better as links and are more clickable.

File: docs/get-started.md [hunk 2]
```diff
@@ -4,27 +4,61 @@ title: Get started
...
+You can use the LiveData Migrator UI or CLI to manage resources and migrate data.
 
-## Get started with the LiveData Migrator UI
+If you want to use the [UI](#get-started-with-the-livedata-migrator-ui) to operate LiveData Migrator, register your LiveData Migrator account after installation. If you want to use the [CLI](#get-started-with-the-livedata-migrator-cli) to operate LiveData Migrator, you're ready to log in.
 
-LiveData Migrator's UI can manage multiple LiveData Migrators as well as LiveData Plane from one convenient interface. It is designed to be intuitive and contains on-screen instructions to help you perform migrations and configuration tasks successfully.
+## LiveData UI
+
+The LiveData UI can manage multiple LiveData Migrators as well as LiveData Plane from one convenient interface. It is designed to be intuitive and contains on-screen instructions to help you perform migrations and configuration tasks successfully.
```
Comment (stepgerner) [inline docs/get-started.md:13]: Multiple LiveData Migrators always feels so weird to me. Customer-facing, shouldn't it be "livedata migrator instances"? If not, I'm not too worried. It's just weird terminology, not even your fault.

File: docs/get-started.md [hunk 3]
```diff
@@ -4,27 +4,61 @@ title: Get started
...
 Internet access from the host is not required, but when it is available, you'll be sent confirmation of your registration. We'll use your registration information to send you important information about your LiveData Migrator account, such as a reminder before your license expires. You can also opt in to receive additional product updates.
 
-## Log in to the LiveData Migrator CLI
+### Add additional LiveData Migrators
+
+If you have additional LiveData Migrator instances, these can be managed in the LiveData UI.
+
+Follow these steps to add an additional instance of LiveData Migrator:
+
+1. On the LiveData UI dashboard, click **+** to add an additional **Products**.
```
Comment (stepgerner) [inline docs/get-started.md:33]: I get what you're going for, but "to add an additional Products" sounds weird. Maybe "click **+** under **Products** to add an additional product".

File: docs/get-started.md [hunk 4]
```diff
@@ -4,27 +4,61 @@ title: Get started
...
+
+You will be taken to the LiveData Migrator **Overview** page once the UI has connected to the instance. The new LiveData Migrator instance will also be displayed on the dashboard in the **Products** section.
+
+### Remove LiveData Migrators
+
+Follow these steps if you want to remove a LiveData Migrator instance from the UI:
+
+1. On the LiveData UI dashboard, click the LiveData Migrator instance that you want to remove in the **Products** section (for example: `LDM-02`).
+1. Click **Configuration -> LiveData Migrator** when viewing the instance.
+1. Click the option to **Remove** the LiveData Migrator (this will also remove the associated HiveMigrator instance as well).
```
Comment (stepgerner) [inline docs/get-started.md:54]: No corrections or anything here, just wanted to point out that these instructions are really clear and informative. I like them.

**Review by stepgerner — COMMENTED** [review]
A few comments.

File: docs/get-started.md (hunk 2 above)
Comment (benradstone) [reply docs/get-started.md:13]: It was how it was originally written, but I was the one who started using "instances". I think I will make it consistent and adjust to "instances" throughout.

**Pull request merged**